transform/snowflake-dbt/models/blp/license_server_fact.sql

```diff
@@ -12,14 +12,75 @@
 
 {% endif %}
 
-with license_union as (
-  SELECT *
-  FROM {{ ref('licensed_servers') }}
-  
+with licensed_servers as (
```

Comment:
Do we need to add `account_mapping` to this file as well?

https://github.com/mattermost/mattermost-data-warehouse/pull/882/files#diff-0e46383f6e4ddebd5d363b6333e4d964e162a9417022aefa3d31799cbc73c560L15-L41

Reply:
```
SELECT
    {{ dbt_utils.surrogate_key(['l.license_id', 'l.server_id']) }} as id
```
We don't need account_mapping as the unique constraint, which is a combination of license_id and server_id, is created after account_mapping is created.

Reply:
@sri-byte Moving your comment over here

> We don't need to add account_mapping since the unique ID is being generated in licensed_servers and nonactivated_licenses. @jasonblais

Got it, thanks for confirming it Sri 👍